go/vt/vttablet/tabletserver/throttle/base/tablet_results.go

```diff
@@ -14,63 +14,42 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// This codebase originates from https://github.com/github/freno, See https://github.com/github/freno/blob/master/LICENSE
```

Comment:
No need for this attribution anymore?

Reply:
Not in this specific file. The code in this file is not derived from `freno`, and the note was needlessly copy+pasted from another file.